cli/src/formatter.rs

```diff
@@ -1274,7 +1272,7 @@ mod tests {
 
         insta::assert_snapshot!(
             str::from_utf8(recorder.data()).unwrap(),
-            @" outer1  inner1  inner2  outer2 ");
+            @" outer1  inner1  inner2  outer2");
```

Comment:
nit: It's not important that the trailing space is omitted here, but it looks a bit odd. We might have to add leading/trailing non-space character to the snapshot sample.

Reply:
You could also file a bug to `insta`.

Reply:
Perhaps a better option is to change the test, so that we in fact are testing everything:

```rust
insta::assert_snapshot!(
            format!("|{}|",str::from_utf8(recorder.data()).unwrap()),
             @"| outer1  inner1  inner2  outer2 |")
```

Reply:
Yes. iirc, insta isn't strict about leading/trailing whitespaces, and I assume that's by design. I just pointed that out because this patch contained whitespace changes like this.